models/annotations_models/train_random_forest_classifier_model.ipynb

```diff
@@ -28,49 +28,116 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 15,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "2024/02/06 10:08:10 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of sklearn. If you encounter errors during autologging, try upgrading / downgrading sklearn to a supported version, or try upgrading MLflow.\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "%matplotlib notebook\n",
     "import pandas as pd\n",
+    "import geopandas as gpd\n",
     "import pickle\n",
     "from sklearn.ensemble import RandomForestClassifier\n",
     "from sklearn.preprocessing import StandardScaler\n",
     "import pprint\n",
+    "import os\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.model_selection import KFold, cross_val_score\n",
+    "from satellite_images_nso_datascience.training.train import train_imbalanced_model, cross_validation_balance_on_date\n",
+    "from satellite_images_nso_datascience.training.utils import get_cross_validation_results_filepath, get_model_filepath\n",
+    "from satellite_images_nso_datascience.training.metric_calculation import calculate_average_metrics, get_metrics\n",
+    "from sklearn.metrics import f1_score\n",
+    "from tif_model_iterator import tif_kernel_iterator\n",
```

Comment:
This import and the one below will break when the package name change in the other repo's goes through.

Reply:
this imports where not needed